ivadomed/models.py

```diff
@@ -488,12 +515,14 @@ def __init__(self, in_channel=1, out_channel=1, depth=3, drop_rate=0.4, bn_momen
         super(Unet, self).__init__()
 
         # Encoder path
-        self.encoder = Encoder(in_channel=in_channel, depth=depth, drop_rate=drop_rate, bn_momentum=bn_momentum,
-                               is_2d=is_2d, n_filters=n_filters)
+        self.encoder = Encoder(in_channel=1, depth=3, drop_rate=0.4, bn_momentum=0.1, n_metadata=None, film_layers=None,
+                 is_2d=True, n_filters=64)
```

Comment:
Why are you hard-coding here? eg we may want to have a multichannel model --> `in_channel` > 1